snapshot_creator.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2021 the v8go contributors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package v8go
+
+// #include <stdlib.h>
+// #include "v8go.h"
+import "C"
+import (
+	"errors"
+	"unsafe"
+)
+
+type FunctionCodeHandling int
+
+const (
+	FunctionCodeHandlingKlear FunctionCodeHandling = iota
```

Comment:
I think this should be `FunctionCodeHandlingClear`?

We should add a comment to each of these explaining what they mean (there should be one that already exists where these two parameters are defined in the v8 header files)